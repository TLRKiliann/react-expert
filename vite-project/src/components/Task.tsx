const Task = (): JSX.Element => {
	return (
		<>
			<h1>Hello Task</h1>
		</>
	)
}

export default Task;