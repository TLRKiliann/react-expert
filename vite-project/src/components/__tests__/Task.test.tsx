import React from "react";
import { render, screen } from "@testing-library/react";
import renderer from 'react-test-renderer';
import { it, expect } from "vitest";
import Task from "../Task";

// toMatchSnapshot
test('renders correctly Task', () => {
  const component = renderer.create(<Task />)
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

