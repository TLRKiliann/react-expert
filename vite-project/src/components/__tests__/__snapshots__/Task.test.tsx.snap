// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renders correctly Task 1`] = `
<h1>
  Hello Task
</h1>
`;
